Model/Schedule.php

```diff
@@ -357,11 +362,37 @@ private function evaluateFileRate($fileRate = 1, $date = null)
     }
 
     /**
-     * @return \MauticPlugin\MauticContactClientBundle\Entity\FileRepository
+     * @return FileRepository
      */
     public function getFileRepository()
     {
-        return $this->em->getRepository('MauticContactClientBundle:File');
+        /** @var FileRepository $repo */
+        $repo = $this->getEntityManager()->getRepository('MauticContactClientBundle:File');
+
+        return $repo;
+    }
+
+    /**
+     * Shore up EntityManager loading, in case there is a flaw in a plugin or campaign handling.
+     *
+     * @return EntityManager
+     */
+    private function getEntityManager()
+    {
+        try {
+            if ($this->em && !$this->em->isOpen()) {
+                $this->em = $this->em->create(
+                    $this->em->getConnection(),
+                    $this->em->getConfiguration(),
+                    $this->em->getEventManager()
+                );
+                // $this->logger->error('ContactClient: EntityManager was closed.');
```

Comment:
Commented out here too

Reply:
Logger isn't available here... Again, this is a bandaid. :(